privacyidea/lib/periodictask.py

```diff
@@ -259,13 +260,14 @@ def get_scheduled_periodic_tasks(node, current_timestamp=None, interval_tzinfo=N
     return scheduled_ptasks
 
 
-def execute_task(taskmodule, params):
+def execute_task(config, taskmodule, params):
```

Comment:
Ah, can we remove the ``config`` parameter again here as well?

Reply:
done in fa76b0b

Reply:
:+1: